### DISCUSSION TITLE
What happened during the week of my absence from the site?

### DISCUSSION THREAD

**Discussion by ARTIK31** [civitai/civitai#418] [Q&A] [answered]
The fact is that I haven't visited the site for a short time, and now what do I see? ALL 18+ content has disappeared somewhere, old loras and checkpoints with adult content have also flown off. I set the settings to show everything, but nothing much has changed.  Tell me plz, maybe I'm doing something wrong or maybe all this content is now in another place? 😱😭

**Comment by TroubleDarkness** [comment]
The only way you can turn off/on NSFW content -> LMB on your avatar in the upper right corner -> Account settings -> Scroll a little bit down -> Content Moderation, now turn on Mature Content and all you want here.

Now you should be able to see "eye" icon on the upper right corner. Set it to "My filters" or "Everything" and remove blur.

![image](https://user-images.githubusercontent.com/33435797/229873881-986f5969-c03d-4411-bc5f-b0f8a79992b1.png)

If this didn't help, then you need help from moderators, i guess.

**Reply by ARTIK31** [reply, marked as answer]
Thanks, as I found it is a bug, so user just should switch this toggle to 'Safe' and back to 'Everything'. After these actions it will work again.
